### PR TITLE
LPS-164774 KB Display widget: select article/folder revamp

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
After the changes: 
Screenshot 2022-10-06 at 13 25 27

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:knowledge-base

**Review by boton — APPROVED** [review]
LGTM!

**Comment by austinchiang** [comment]
**Reviewing...** :eyes:

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp [hunk 1]
```diff
@@ -58,7 +58,7 @@ kbSelectParentDisplayContext.populatePortletBreadcrumbEntries(currentURLObj);
 							"title", parentTitle
 						).build()
 					%>'
-					value='<%= (kbSelectParentDisplayContext.getParentResourceClassNameId() == kbFolderClassNameId) ? "choose-this-folder" : "choose-this-article" %>'
+					value='<%= (kbSelectParentDisplayContext.getParentResourceClassNameId() == kbFolderClassNameId) ? "select-this-folder" : "select-this-article" %>'
```
Comment (austinchiang) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp:63]: FYI bookmarks and recycle bin still use "Choose This Folder." I am not sure if those needed to be updated (if so they will break tests)

**Comment by austinchiang** [comment]
Resent in https://github.com/liferay-lima/liferay-portal/pull/3571

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp:63]: I don't think those should be updated for now...

**Pull request closed without merging**